o2a/converter/workflow.py

```diff
@@ -50,8 +50,8 @@ def __init__(
         self.dependencies = dependencies or {
             "import shlex",
             "import datetime",
-            "from o2a.o2a_libs.property_utils import PropertySet",
-            "from o2a.o2a_libs import functions",
+            "from o2a_lib.property_utils import PropertySet",
```

Comment:
in other places we add .src. - should it be the case here too?

Reply:
No, because dependencies in this case are for the result file which the users get after conversion. And for this result file we import functions from o2a_lib library. src path needed for internal usage for o2a library.